src/guides/v2.4/release-notes/release-notes-2-4-0-open-source.md

```diff
@@ -5,6 +5,8 @@ title: Magento Open Source 2.4.0 Release Notes
 
 Magento Commerce 2.4.0 introduces support for PHP 7.4, Elasticsearch 7.6.x, and MySQL 8.0. Substantial security changes include the enablement of two-factor authentication in the Admin by default.
 
+**With this release, the Authorize.Net, eWay, CyberSource, and Worldpay payment method integrations have been removed from core code. Merchants should migrate to the official extensions that are available on the Magento Marketplace**.
```

Comment:
Authorize.Net is the only one of these available in Open Source.

Reply:
PM asserts that the original text is correct.